llvm_mode/afl-clang-fast.c

```diff
@@ -685,9 +715,22 @@ int main(int argc, char **argv, char **envp) {
 
 #ifndef AFL_CLANG_FLTO
   if (instrument_mode == INSTRUMENT_LTO)
-    FATAL("instrumentation mode LTO specified but LLVM support not available");
+    FATAL(
+        "instrumentation mode LTO specified but LLVM support not available "
+        "(requires LLVM 11)");
```

Comment:
'11' -> '11 or above'

Reply:
there is only 11 so far, but yeah lets be future proof. but we dont know if it will work with llvm 12 ..